src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -137,11 +137,11 @@ SELECT function(attribute)
       <td id="facet">
         `FACET attribute`
 
-        **Required for outlier conditions, but not baseline or static**
+        **Required for outlier conditions.**
       </td>
 
       <td>
-        Include an optional `FACET` clause in your NRQL syntax depending on the [threshold type](#threshold-types): static, baseline, or outlier.
+        Include an optional `FACET` clause in your NRQL syntax depending on the [threshold type](#threshold-types): static or baseline.
 
         Use the [`FACET`](/docs/insights/nrql-new-relic-query-language/nrql-resources/nrql-syntax-components-functions) clause to separate your results by attribute and alert on each attribute independently. No `LIMIT` clause is allowed, but all queries will receive the maximum number of facets possible. Faceted queries can return a maximum of 5000 values for [static](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries#threshold-types) conditions and a maximum of 500 values for [outlier](/docs/alerts/new-relic-alerts/defining-conditions/outlier-detection-nrql-alert) conditions.
```

Comment:
`Faceted queries can return a maximum of 5000 values for static and baseline conditions`